tensorflow/c/experimental/gradients/math_grad.cc

```diff
@@ -31,22 +34,31 @@ class AddGradientFunction : public GradientFunction {
  public:
   Status Compute(Context* ctx,
                  absl::Span<AbstractTensorHandle* const> grad_inputs,
-                 vector<AbstractTensorHandle*>* grad_outputs) override {
+                 std::vector<AbstractTensorHandle*>* grad_outputs) override {
```

Comment:
We should not need the `std::` here because we have the `using std::vector` declaration earlier. Please fix other usages as well.

Reply:
SGTM